pom.xml

```diff
@@ -17,7 +17,7 @@
     <url>https://github.com/folio-org/edge-oai-pmh.git</url>
     <connection>scm:git:git://github.com/folio-org/edge-oai-pmh.git</connection>
     <developerConnection>scm:git:git@github.com:folio-org/edge-oai-pmh.git</developerConnection>
-    <tag>v1.0.0</tag>
+    <tag>v2.0.0</tag>
```

Comment:
I'm pretty sure this should be `HEAD`.  The only exception would be in your release tag.